### PR TITLE
Change doc theme to furo

### PR DESCRIPTION
**Pull request by hofaflo**
Preview for the discussion in #73

### CODE REVIEW

**Comment by cbrnr** [comment]
This looks really nice! I like it much better than the previous theme. One minor comment, can you move the API sentence down on the main page as well? Otherwise LGTM!

**Comment by cbrnr** [comment]
And maybe the title could just be "SleepECG" without "documentation"?

File: doc/source/index.md [hunk 1]
```diff
@@ -2,12 +2,11 @@
    :hidden:
 
    Home <self>
+   API <api>
```
Comment (cbrnr) [inline doc/source/index.md:5]: Why this change?

Comment (hofaflo) [reply doc/source/index.md]: good question, that shouldn't have happened 🤔

**Comment by cbrnr** [comment]
Ready to merge?

**Comment by hofaflo** [comment]
yes!

**Comment by cbrnr** [comment]
Thanks @hofaflo!
